src/App.tsx

```diff
@@ -21,14 +38,18 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos ? (
+                <TodoList />
```

Comment:
Replace the ternary operator with conditional rendering using '&&' for better readability. For example: `{todos && <TodoList />} { !todos && <Loader />}`